.env.example

```diff
@@ -0,0 +1,2 @@
+We will be using this folder to store all our app, api and secret keys. 
```

Comment:
For this one... I think we should use .env.local
Also the .env files should be listed in .gitignore

Reply:
I have changed it to env.local, seems like they stopped allowing the usage of only .env